toolsets/nodejs/npm-install-global-deps

```diff
@@ -1,22 +1,22 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
This script previously did not have the `-e` flag set in the prelude like most others do. I added it in this PR to address [Linter warning SC2164](https://www.shellcheck.net/wiki/SC2164) (performing a `cd` without checking if it succeeded/failed). Is there a reason to not include this flag for this utility script?

Reply:
Sometime there's a reason... but... most likely its an omission and if there's a valid reason there should be a comment. If there's a reason you'll see set -e or set -ex lower in the script.

The main reason -e is omitted is because it silently terminates so -e alone is quite rare, you'd have -ex most of the time.